clients/hadoopfs/src/main/java/io/lakefs/committer/DummyOutputCommitter.java

```diff
@@ -33,7 +35,10 @@ public DummyOutputCommitter(Path outputPath, JobContext context) throws IOExcept
         // TODO(lynn): Create lakeFS API client.
 
         if (outputPath != null) {
-            FileSystem fs = outputPath.getFileSystem(context.getConfiguration());
+            Configuration conf = context.getConfiguration();
+            FileSystem fs = outputPath.getFileSystem(conf);
+            LakeFSClient lakeFSClient = new LakeFSClient(fs.getScheme(), conf)
+
             Preconditions.checkArgument(fs instanceof LakeFSFileSystem,
                                         "%s not on a LakeFSFileSystem", outputPath);
```

Comment:
No point in creating a client if the filesystem is incorrect.  Since creation might just throw (e.g. if `fs.getScheme()` returns an unknown scheme), I prefer to verify the filesystem first:

```suggestion
            Preconditions.checkArgument(fs instanceof LakeFSFileSystem,
                                        "%s not on a LakeFSFileSystem", outputPath);
            LakeFSClient lakeFSClient = new LakeFSClient(fs.getScheme(), conf)
```